rust-toolchain

```diff
@@ -1 +1 @@
-1.29.0
```

Comment:
Would it make sense to set this to "stable"?

Reply:
I had meant to commit this to master -- oops!

I've often set the toolchain to `stable` in the past, but a few times I've seen machines that haven't run `rustup update` in a while and the "stable" toolchain is behind what I'm actually using. It's a bit of a confusing error to deal with, but perhaps that's not enough of an issue to justify the clerical work.